### PR TITLE
Feature/#34/combine config parameters

### PR DESCRIPTION
**Pull request by felixble**
Related issues: #34 and #9

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,10 +28,10 @@ before_install:
...
-  - cp local_travis.php local.php
-  - cp ./test/util/dbConfigTravis.json ./test/util/dbConfig.json
+  - cp .env.travis .env
   - composer install
   - ./travis-setup-db.sh
 
 script:
-  - testcafe chrome,firefox test/**/*.test.js --app "php -S 0.0.0.0:8888 -t ./"
\ No newline at end of file
+  - testcafe chrome,firefox test/**/*.test.js --app "php -S 0.0.0.0:8888 -t ./"
```
Comment (jasteuer) [inline .travis.yml]: is it possible to use one of the npm scripts (`npm run test:e2e:XXX`) here?

File: test/e2e/run.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 cd "$(dirname "$0")"
 
-## start the php server 
+## start the php server
 # php -S localhost:8888 -t ../../ &
 # SERVER_PID=$!
```
Comment (jasteuer) [inline test/e2e/run.sh]: I think this file is no longer necessary:
- developer starts server with vagrant and use npm scripts to run tests
- travis starts server and run tests according to its configuration

File: .travis.yml (hunk 1 above)
Comment (jasteuer) [inline .travis.yml]: is it possible to start the server and then use one of the npm scripts (`npm run test:e2e:XXX`) here?

**Review by jasteuer — CHANGES_REQUESTED** [review]
Looks good! Only small things I requested to change

File: test/e2e/run.sh (hunk 2 above)
Comment (felixble) [reply test/e2e/run.sh]: Yep, you are right. I will remove it

File: .travis.yml (hunk 1 above)
Comment (felixble) [reply .travis.yml]: Good idea. Thanks!

**Comment by felixble** [comment]
@jasteuer Thanks for your review. I have implemented the requested changes. Please do not forget to append "Closes #34 and closes #9" to the commit message when merging this PR since both tickets are done with this PR.

**Pull request merged**